integration-tests/tests/vexination.rs

```diff
@@ -194,10 +194,10 @@ async fn test_upload_vex_empty_json(context: &mut VexinationContext) {
 
 #[test_context(VexinationContext)]
 #[tokio::test]
-#[ntest::timeout(60_000)]
+#[ntest::timeout(90_000)]
 async fn test_upload_vex_empty_file(vexination: &mut VexinationContext) {
     let id = "empty-file-upload";
-    wait_for_event(Duration::from_secs(30), &vexination.config, "vex-failed", id, async {
+    wait_for_event(Duration::from_secs(60), &vexination.config, "vex-failed", id, async {
```

Comment:
I'm curious why this is necessary?

Reply:
It was failing in my initial PR build, and it looked like the timeout happened just before the indexer printed an error parsing the document. So either:

1) The timeout was to low
2) The indexer thread/task was blocked from running

Increasing the timeout should rule out option 1 at least.